src/sourmash/command_sketch.py

```diff
@@ -50,10 +56,16 @@ def _parse_params_str(params_str):
                 scaled = int(scaled)
             except ValueError:
                 raise ValueError(f"cannot parse scaled='{scaled}' as an integer")
+            # if scaled < 0:
+            #     raise ValueError(f"scaled is {scaled}, must be >= 1")
+            # if scaled > 1e8:
+            #     notify(f"WARNING: scaled value of {scaled} is nonsensical!?")
             if scaled < 0:
-                raise ValueError(f"scaled is {scaled}, must be >= 1")
-            if scaled > 1e8:
-                notify(f"WARNING: scaled value of {scaled} is nonsensical!?")
+                raise ValueError(f"ERROR: --scaled value must be positive")
```

Comment:
same point here -- --scaled isn't used here so these error messages are misleading.

Reply:
I'm not sure I understand what you mean. Should it be `scaled` instead of `--scaled` and `num` instead of `--num`?